mario/tools/tabletransform.py

```diff
@@ -232,3 +232,48 @@ def SUT_to_IOT(instance, method):
     rename_index(matrices["baseline"])
 
     return matrices, indeces, units
+
+
+def ISARD_TO_CHENERY_MOSES(instance,scenario):
+
+    regions = instance.get_index(_MASTER_INDEX['r'])
+    commodities = instance.get_index(_MASTER_INDEX['c'])
+    sN = slice(None)
+
+    U_isard = instance.get_data(['U'], scenarios=[scenario])[scenario][0]
```

Comment:
I would suggest to use the query function isntead of get_data, as it works exactly the same, but you dont need to take the scenario and position anymore when a single scneario and matrix is passed:
```

instance.get_data(['U'], scenarios=[scenario])[scenario][0]

# is the equivalent of

instance.query(_ENUM.U,scenarios=scenario)
```

Reply:
I tried to change to query, but it seems U and S are not acceptable matrices